.ci/docker/test/operate-docker-labels.golden.json

```diff
@@ -6,7 +6,7 @@
   "io.openshift.wants": "zeebe,elasticsearch",
   "io.openshift.tags": "bpmn,operate,camunda",
   "org.opencontainers.image.authors": "operate@camunda.com",
-  "org.opencontainers.image.base.name": $BASE_IMAGE,
+  "org.opencontainers.image.base.name": "docker.io/library/alpine:3.19.1",
```

Comment:
Hard-coding this here will get outdated when Renovate updates the base image, because Renovate doesn't know about the JSON format in goldenfiles - I think?

Reply:
So before it was in the [job run part of the workflow](https://github.com/camunda/zeebe/blame/main/.github/workflows/operate-release-reusable.yml#L233), I think it had to be updated manually before as well. 

Example update https://github.com/camunda/zeebe/commit/5a232db58ce22d0125bdf77713c85393c6476f29

As this will fail the verification we should any will release in the corresponding renovate PR

Reply:
Maybe @johanwelgemoed or @ralfpuchert know best if manual updates where always needed, if so than not a regression :+1:

Reply:
I am all for not having things hardcoded and requiring manual updates so I think this should probably work the same across all components. @ralfpuchert do you agree? I cannot see a reason why we need this hardcoded.

Reply:
> @cmur2 maybe you could propose a different way if you think it is a problem, so we don't have to wait here

Sorry I don't have capacity right now to do this, just highlighting problems that I see and referring to the owners/experts who should know.

Reply:
I see right no good or better way for this. As it is the same as before I would see this as a blocker for the PR.

To improve the situation at least a bit I would go ahead and add at least the docker verification to the Operate CI. 

If you have other solutions please make a proposal or/and create an issue to improve it.

Reply:
> I see right no good or better way for this. As it is the same as before I would see this as a blocker for the PR.

Edit: I of course meant:  **I wouldn't see this as a blocker for the PR.**

Reply:
For me to fully understand:
This is not a regression but a manual update of the base image was needed already (just in the workflow file where the [BASE_IMAGE var originated](https://github.com/camunda/zeebe/commit/5a232db58ce22d0125bdf77713c85393c6476f29)) if renovate updated the image, right?

So all that changes with this PR is the location of the value that needs manual treatment?

Reply:
That is correct.

Reply:
I had a look at the Operate CI (or better say the [reusable workflow](https://github.com/camunda/zeebe/blob/main/.github/workflows/operate-ci-build-reusable.yml)). I feel it is not that easy to insert the verification without a bigger refactoring of the workflow. 

The workflow seems to build images, but also push them to the camunda registry? On `main` building SNAPSHOT and pushing them to docker hub. 

The verification can be done afterward building (and pushing) only, not sure how useful this is when we already pushed the image. In the [Zeebe ci for example we use a local registry](https://github.com/camunda/zeebe/blob/main/.github/workflows/zeebe-ci.yml#L431), to a) not push unnecessary b) only push images, if it is valid.

As we also plan to unify the CI I would expect this might can be also done there \cc @cmur2 (might be otherwise unnecessary double work and bloat this PR even more + it is the same as before)